src/picongpu/include/plugins/adios/restart/LoadParticleAttributesFromADIOS.hpp

```diff
@@ -82,8 +83,9 @@ struct LoadParticleAttributesFromADIOS
 
         for (uint32_t n = 0; n < components; ++n)
         {
+            OpenPMDName<T_Identifier> openPMDName;
             std::stringstream datasetName;
-            datasetName << particlePath << T_Identifier::getName();
+            datasetName << particlePath << openPMDName();
```

Comment:
Why did you name the prefix variable `particlePath` (and not `subGroup`) as in hdf5?